rmw_connextdds_common/src/common/rmw_context.cpp

```diff
@@ -218,6 +238,24 @@ rmw_context_impl_t::initialize_node(
     return RMW_RET_ERROR;
   }
 
+  if (this->qos_library != "") {
+    struct DDS_StringSeq library_names;
+    DDS_StringSeq_initialize(&library_names);
+    if (
+      DDS_RETCODE_OK != DDS_DomainParticipantFactory_get_qos_profile_libraries(
```

Comment:
This function is only available in RTI Connext DDS Pro, and not in Micro. For this reason, we can either:
- wrap this call in a function in `dds_api.hpp` (e.g. `rmw_connextdds_get_qos_profile_libraries(ctx, libraries)`) which can then be a no-op for Micro.
- Add the prototype for `DDS_DomainParticipantFactory_get_qos_profile_libraries` to `dds_api_rtime.hpp` and empty implementation for it in `dds_api_rtime.cpp` (or it could even be a `#define` to `DDS_RETCODE_OK`).

Reply:
I don't have a copy of RTI Connext micro to test, but I think https://github.com/rticommunity/rmw_connextdds/pull/7/commits/bcc37d259e924bafe0ea885290a6a1155b8df291 will fix it.
Let me know if something else is needed.